src/commands/push-to-app-collection.yaml

```diff
@@ -0,0 +1,45 @@
+parameters:
+  app_name:
+    type: "string"
+  app_catalog:
+    type: "string"
+  app_collection_repo:
+    type: "string"
+  docker_username_envar:
+    type: "string"
+  docker_password_envar:
+    type: "string"
+steps:
+  - run:
+      name: get project version
+      command: |
+        architect project version > version
+        echo -n "<<parameters.app_name>>-$(cat version)" | head -c 53 > cr-name
```

Comment:
We need this because of helm limitation of 53 characters.

Reply:
I wonder if it shouldn't fail instead of truncating.

Reply:
This job should only push tagged versions, so it's very unlikely that we hit the 53 chars limit. But I needed that for testing.

Reply:
I would personally prefer to fail instead. If this becomes a problem then at least we know the cause and can decided on a solution having that knowledge.